run.py

```diff
@@ -35,29 +40,51 @@ def create_app(config):
     return app
 
 
+def create_database(db_connection, db_schema):
+    from ras_party.models import models
+
+    def current_request():
+        return _app_ctx_stack.__ident_func__()
+
+    engine = create_engine(db_connection, convert_unicode=True)
+    session = scoped_session(sessionmaker(), scopefunc=current_request)
```

Comment:
If we're using scoped_session do we need the `with_db_session` stuff?

Reply:
Can. Of. Worms. But yeah, probably beyond this PR though.